docs/embedding/introduction.md

```diff
@@ -1,165 +1,39 @@
 ---
-title: Embedding Metabase in other applications
+title: Introduction
 redirect_from:
   - /docs/latest/administration-guide/13-embedding
-  - /docs/latest/administration-guide/12-public-links
 ---
 
-# Embedding Metabase in other applications
+# Introduction
 
-Metabase includes a powerful application embedding feature that allows you to embed your saved questions or dashboards in your own web applications. You can even pass parameters to these embeds to customize them for different users.
+Embedding allows people to view your Metabase app inside another web app (such as your company's website).
 
-## Key Concepts
+[Signed embedding](./signed-embedding) and [full-app embedding](./full-app-embedding) are _secure_ ways to share your data with specific groups of people outside of your organization.
 
-### Applications
+If you'd like to share your data with the good people of the internet, you can create a [public link](../questions/sharing/public-links) and embed that directly on your website.
 
-An important distinction to keep in mind is the difference between Metabase and the embedding application. The charts and dashboards you will be embedding live in the Metabase application, and will be embedded in your application (i.e. the embedding application).
+## How embedding works
 
-### Parameters
+You'll need to put an iframe on your website to act as a window to your Metabase app. Then, you can configure things like:
 
-Some dashboards and questions have the ability to accept [parameters](../questions/native-editor/sql-parameters.md). In dashboards, these are synonymous with [dashboard filters](../dashboards/filters.md). For example, if you have a dashboard with a filter on `Publisher ID`, this can be specified as a parameter when embedding, so that you could insert the dashboard filtered down to a specific `Publisher ID`.
+- what Metabase charts and dashboards to display in the iframe,
+- whether people need to sign in to view those charts and dashboards, and
+- how much people can interact with your data (using Metabase features that are accessible through the iframe).
 
```

Comment:
I like this list. We should add links to the respective types here as well (for some useful redundancy with the links in the table).

Reply:
I still think we need a list of the three embedding types here, with links to the relevant docs. It took me a sec to realize I could click on the column headings in the table below, so I'm assuming some people will stumble here as well.